web/client/reducers/layers.js

```diff
@@ -102,15 +102,24 @@ function layers(state = [], action) {
             });
             return assign({}, state, {refreshing: newLayers});
         }
+        case CHANGE_LAYER_PARAMS:
```

Comment:
So, is CHANGE_LAYER_PARAMS just an alias of CHANGE_LAYER_PROPERTIES ?

Reply:
Yes, the action creators are different in terms of entries they add to the action object, and this is the reason of different . 
I don't know if it is easier to use the same action name with 2 different action creators or use 2 different names. Do you think I should change it?